packages/network-of-terms-catalog/catalog/datasets/gtaa-classificatie.jsonld

```diff
@@ -30,6 +30,13 @@
         {
           "@type": "FindAction",
           "query": "file://catalog/queries/lookup/gtaa.rq"
+        },
+        {
+          "@type": "Action",
+          "actionApplication": {
+            "@id": "https://reconciliation-api.github.io/specs/latest/",
+            "@type": "SoftwareApplication"
+          }

```

Comment:
@wmelder This will add support for Reconciliation Service for GTAA datasets in the Network of Terms. Please let me know your new GTAA infrastructure already supports the [Reconciliation Service API](https://reconciliation-api.github.io/specs/latest/). In that case the Network of Terms should refer to that instead.

Reply:
@ddeboer This is the first time that I see this service specification, so I don't expect this to be supported by the GTAA service layer.

Reply:
To add some context: the Reconciliation spec is used (mostly) by OpenRefine to match strings against things (in our case, URIs of terms). 

No problem if GTAA doesn’t have this. In that case the Network of Terms provides this for you, so users can still reconcile their data with terms from the GTAA.

Reply:
ok, that's great!